data/campaigns/atl02.wmf/scripting/map_editing.lua

```diff
@@ -0,0 +1,156 @@
+function volcano_eruptions()
+   local fields_to_erupt = Set:new{}
+   local fields_to_setback = Set:new{}
+   local central_field = map:get_field(133,47)
+   local region_to_erupt = central_field:region(17)
+   for a,field in next,region_to_erupt,f do
```

Comment:
`f` is uninitialised here. (I used a variable `f` for fields in the debug console and I got a lua coroutine error when the test game got to this point.)

Is using `next()` somehow different from `ipairs()`? As far as I can tell the effect is the same.

```suggestion
   for a,field in ipairs(region_to_erupt) do
```